package.json

```diff
@@ -10,12 +10,14 @@
     "harp:compile": "npm run harp compile styleguide dist",
     "start": "harp server styleguide",
     "gh-pages": "./scripts/build-pages.sh",
-    "preversion": "npm run harp compile styleguide dist && git add -f dist",
+    "preversion": "npm run harp compile styleguide dist && npm run babel && git add -f dist",
     "postversion": "git push && git push --tags",
     "sass:compile": "node-sass --output-style=compressed --quiet -o /tmp styleguide/css",
     "sass-lint": "sass-lint -v",
     "hooks": "./scripts/hooks.sh",
-    "postinstall": "npm run hooks"
+    "postinstall": "npm run hooks",
+    "js-lint": "eslint styleguide/js/zoolander.js",
```

Comment:
we are going to need to add a `--max-warnings=#` here and then also add `npm run js-lint` to the `hooks` file.   That said, I'm seeing actual errors, not just warnings so we may need to figure out how to handle that first.